src/cli/commands/test/mod.rs

```diff
@@ -291,3 +292,41 @@ impl CommandExecution<TestOutput, TestCommandError> for TestArgs {
 		Ok(Default::default())
 	}
 }
+
```

Comment:
Put it in a separated file

Reply:
Any suggestion for the file name ?

Reply:
maybe we can call it `program_parsing.rs`, so it will have a larger purpose. Ready to host more functionalities

Reply:
Ok I put it in a separated file.